### ISSUE TITLE
Screen brightness value been reset every time enter or leave a game

### ISSUE THREAD

**Issue by KyleBing** [spruceUI/spruceOS#27]
| Item | Version|
|---|---|
|SpruceOS | v2.1 |
|Firmware| 20240713 |

It seems the brightness has been reset to default value every time I enter or leave a game, or turn on the device.

Another thing:

Let's say the brightness value is 1-12. For example, when I turn on the device, the brightness has been set to 6. 
The value stored in the user profile is 2, where I left it last time to turn off the device.

So:

```bash
      2/12                    6/12   
    should be            current state
```

But when I use START + L | R  to adjust it, it changes from the stored value 2 which I set last time, instead of the current state 6.

So in summary, the bug would be the user brightness value has not been applied every time the user enters or leaves a game, either turning on the device.
